orca-core/orca-core.gradle

```diff
@@ -40,6 +40,8 @@ dependencies {
   compile "com.fasterxml.jackson.module:jackson-module-kotlin:${spinnaker.version('jackson')}"
   compile "org.apache.commons:commons-lang3:3.7"
   compile "de.huxhorn.sulky:de.huxhorn.sulky.ulid:8.1.1"
+  compile "org.springframework.security:spring-security-web:4.2.10.RELEASE"
```

Comment:
This can be `compile spinnaker.dependency("springSecurityWeb")` which will pin it to our Spinnaker-wide dependency.